### ISSUE TITLE
Can't enter text in input that uses a html5 input type

### ISSUE THREAD

**Issue by nileshtochia** [stirno/FluentAutomation#106]
Throws FluentException ""Enter().In() is only supported on text elements (input, textarea, etc)." when attempting to use Enter().In() on an input that use html5 types such as email.

**Comment by pauljz** [comment]
Confirmed that this is broken in 3.0. I'll try to get a fix in today if @stirno doesn't get to it first.

**Comment by pauljz** [comment]
Oops accidentally failed at mobile interfaces. Didn't mean to close that.

**Comment by stirno** [comment]
Accepted a PR towards this issue from that didn't get tied to this.. https://github.com/pauljz/FluentAutomation/commit/2c999391b37914cb6c83fa0e170abb0be1622042

For now we've added support for fields where we explicitly know the UX will remain text-based data entry.  The newly supported fields are email, search, url, tel and number. I expect as more browsers implement these field types we'll be able to better develop DSLs for interacting with them.

If a user needs to do more with the other fields types, there is always the Focus and Press/Type commands to get a bit lower level.

Closing as resolved for now - will make it to NuGet in the next push. In dev branch on GH now.
